docs/pages/model-context-protocol/mcp-access.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: MCP Access
```

Comment:
"MCP Access" sounds nebulous; the guide discusses MCP clients, so maybe rename to something like this? Not quite sure what proper title should be.

Reply:
What do you think putting these under User Guides alongside with like database gui clients, and Teleport Connect? We can combine them into one guide too like `MCP Clients`.

Reply:
I'll move them, but keep the "root" link too, as per the comment on the other PR.

I don't see any problem with combining them. I want to ensure both have the same visibility, and having a larger single page (similar to the [Database Access through WebUI](http://localhost:3000/ver/19.x/connect-your-client/web-ui/#starting-a-database-session)) might not provide enough visibility for the guide that comes second. What do you think?